### PR TITLE
[RSDK-4743] Add warning for connecting jetson pins to signals over 3.3V

### PR DESCRIPTION
**Pull request by randhid**
Hidden deep within jetson documentation is a spec that says the pins "all operate on 3.3v signals". This means that you can harm them by attaching signals over 3.3v, such as from an encoder powered by 5v. Here is my attempt at a warning to users so they don't fry their pins.

### CODE REVIEW

File: docs/components/board/jetson.md [hunk 1]
```diff
@@ -15,6 +15,12 @@ Follow one of our Jetson [setup guides](/installation/) to prepare your board fo
...
 
+{{% alert title="REQUIREMENTS" color="warning" %}}
+
+The jetson's gpio pins are rated for inputs and outputs at 3.3V. Signals from encoders and sensors at even 5V can cause damage to a pin. We recommend connecting hardware that can operate and send signals at 3.3V or lower. [Jetson Nano Developer Kit 40-Pin Expansion Header GPIO Usage Considerations Applications Note](https://developer.nvidia.com/jetson-nano-developer-kit-40-pin-expansion-header-gpio-usage-considerations-applications-note)
+
+{{% /alert %}}
+
 Configure a `jetson` board to integrate a [NVIDIA Jetson Orin Module and Developer Kit](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-orin/), [NVIDIA Jetson Xavier NX](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-agx-xavier/), or [NVIDIA Jetson Nano](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-nano/) into your robot:
 
 {{< tabs name="Configure an jetson Board" >}}
```
Comment (penguinland) [inline docs/components/board/jetson.md:26]: Unrelated to your change, but while you're in here: make it "a jetson" and not "an jetson"

File: docs/components/board/jetson.md [hunk 2]
```diff
@@ -15,6 +15,12 @@ Follow one of our Jetson [setup guides](/installation/) to prepare your board fo
 
 {{% /alert %}}
 
+{{% alert title="REQUIREMENTS" color="warning" %}}
+
+The jetson's gpio pins are rated for inputs and outputs at 3.3V. Signals from encoders and sensors at even 5V can cause damage to a pin. We recommend connecting hardware that can operate and send signals at 3.3V or lower. [Jetson Nano Developer Kit 40-Pin Expansion Header GPIO Usage Considerations Applications Note](https://developer.nvidia.com/jetson-nano-developer-kit-40-pin-expansion-header-gpio-usage-considerations-applications-note)
```
Comment (penguinland) [inline docs/components/board/jetson.md:20]: At the beginning of the paragraph, gpio is not capitalized, but at the end, GPIO is. Change one or the other so it's consistent.

Comment (penguinland) [reply docs/components/board/jetson.md:20]: I clicked the link, and NVidia prompted me to create an account and log in before I could read whatever it is. After creating an account, I was shown a 12-page PDF, with no obvious indication of which part of the PDF you're trying to refer to. Is this link worth having at all?

Comment (penguinland) [reply docs/components/board/jetson.md:20]: If you're going to keep the link, consider rephrasing it into a sentence: `For details, see page XX of the [Jetson Nano Developer Kit...](...)`

**Review by penguinland — APPROVED** [review]
It could use a tweak or two, but the big picture LGTM. Thanks!

File: docs/components/board/jetson.md (hunk 1 above)
Comment (randhid) [reply docs/components/board/jetson.md:26]: not going to edit anything else in this file.

File: docs/components/board/jetson.md (hunk 2 above)
Comment (randhid) [reply docs/components/board/jetson.md:20]: I did want to ask @npentrel about including the page number. Seeed studio uploaded a version of this that you can access without an account, but I didn't want to link the unofficial documentation. I think the link is worth having if people want to read about other gpio considerations.

File: docs/components/board/jetson.md [hunk 3]
```diff
@@ -15,6 +15,12 @@ Follow one of our Jetson [setup guides](/installation/) to prepare your board fo
 
 {{% /alert %}}
 
+{{% alert title="REQUIREMENTS" color="warning" %}}
```
Comment (npentrel) [inline docs/components/board/jetson.md:18]:
```suggestion
{{% alert title="CAUTION: Use 3.3V inputs and outputs" color="warning" %}}
```

**Review by npentrel — APPROVED** [review]
Thank you for this change - please accept the suggestion (we try to standardize these note callouts) and then feel free to merge!

File: docs/components/board/jetson.md (hunk 2 above)
Comment (randhid) [reply docs/components/board/jetson.md:20]:
```suggestion
The jetson's GPIO pins are rated for inputs and outputs at 3.3V. Signals from encoders and sensors at even 5V can cause damage to a pin. We recommend connecting hardware that can operate and send signals at 3.3V or lower. For details, see pages 1-3 of the [Jetson Nano Developer Kit 40-Pin Expansion Header GPIO Usage Considerations Applications Note](https://developer.nvidia.com/jetson-nano-developer-kit-40-pin-expansion-header-gpio-usage-considerations-applications-note)
```

**Pull request merged**